data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSource.java

```diff
@@ -53,9 +54,14 @@ public void start(Buffer<Record<String>> buffer) {
             final int threads = sourceConfig.getThreadCount();
             final ScheduledThreadPoolExecutor blockingTaskExecutor = new ScheduledThreadPoolExecutor(threads);
             sb.blockingTaskExecutor(blockingTaskExecutor, true);
-            // TODO: attach ThrottlingService
+            final int maxPendingRequests = sourceConfig.getMaxPendingRequests();
+            final LogThrottlingStrategy logThrottlingStrategy = new LogThrottlingStrategy(
+                    maxPendingRequests, blockingTaskExecutor.getQueue());
+            final LogThrottlingRejectHandler logThrottlingRejectHandler = new LogThrottlingRejectHandler(maxPendingRequests);
+            // TODO: allow customization on URI path for log ingestion
+            sb.decorator("/log/ingest", ThrottlingService.newDecorator(logThrottlingStrategy, logThrottlingRejectHandler));
             final LogHTTPService logHTTPService = new LogHTTPService(requestTimeoutInMillis, buffer);
-            sb.annotatedService(logHTTPService);
+            sb.annotatedService("/log/ingest", logHTTPService);
```

Comment:
This is the second time this string is used. I would recommend defining a constant for "/log/ingest".

Reply:
I think we eventually want this to be a configuration. It might be good to just make that change now.

Reply:
@dlvenable Upon a second thought, allow customizing the URI path for log ingestion seems not adding value to the source plugin despite that FluentBit HTTP output plugin allows configuring custom URI path. We might even just replace this URI path with root path "/".

For now I will make it a constant and we could deal with this minor change in later cleanup PRs.